controllers/UserController.php

```diff
@@ -45,6 +50,11 @@ public function actions()
         ];
     }
 
+    public function init()
+    {
```

Comment:
[Call the parent implementation of this method ](https://www.yiiframework.com/doc/guide/2.0/en/concept-components)